### ISSUE TITLE
develop branch: scide segfaults on startup

### ISSUE THREAD

**Issue by shimpe** [supercollider/supercollider#4064]
Environment
-----------
* Your SuperCollider version: develop branch sha1 78092d1e2eeb41065e65f02a80715c4c44bb173a

* Your operating system and version: arch linux 64 bit 4.18.6-arch1-1-ARCH #1 SMP PREEMPT Wed Sep 5 11:54:09 UTC 2018 x86_64 GNU/Linux
* qt 5.11.1, gcc 8.2.0, 64bit

Steps to reproduce (for bugs)
-----------------------------

~/development/supercollider/build]$ scide

Error message (for bugs)
------------------------

Failed to open web socket server.
Received signal 6
#0 0x7fca0d2a289e <unknown>
#1 0x7fca0d2a29b0 <unknown>
#2 0x7fca0d2a2fe7 <unknown>
#3 0x7fca120ba3c0 <unknown>
#4 0x7fca09965d7f __GI_raise
#5 0x7fca09950672 __GI_abort
#6 0x7fca09ea27fc QMessageLogger::fatal()
#7 0x55d2d3560d06 main.cold.79
#8 0x7fca09952223 __libc_start_main
#9 0x55d2d3573cfe _start
  r8: 0000000000000000  r9: 00007ffd95dcffd0 r10: 0000000000000008 r11: 0000000000000246
 r12: 00007ffd95dd0400 r13: 0000000000000000 r14: 000055d2d51bae10 r15: 00007ffd95dd0480
  di: 0000000000000002  si: 00007ffd95dcffd0  bp: 00007ffd95dd0510  bx: 0000000000000006
  dx: 0000000000000000  ax: 0000000000000000  cx: 00007fca09965d7f  sp: 00007ffd95dcffd0
  ip: 00007fca09965d7f efl: 0000000000000246 cgf: 002b000000000033 erf: 0000000000000000
 trp: 0000000000000000 msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.

**Comment by shimpe** [comment]
Nevermind, I cleaned everything, removed all traces of supercollider from my system, rebuilt, reinstalled and now it seems to work normally....

**Comment by cancandan** [comment]
I am having this problem, ubuntustudio 19.04
QApplication: invalid style override passed, ignoring it.
Failed to open web socket server.
Aborted (core dumped)
[1:1:0100/000000.802255:ERROR:broker_posix.cc(41)] Invalid node channel message

**Comment by snappizz** [comment]
@cancandan hi, are you using the latest version in the 3.10 branch?

**Comment by cancandan** [comment]
@snappizz I tried with the develop branch. Thanks I will try again.

**Comment by snappizz** [comment]
if its still a problem, please let me know by opening a new issue. would be good to have a crash report (see [Generating Crash Reports](https://github.com/supercollider/supercollider/wiki/Generating-crash-reports) for info)

**Comment by philbusby** [comment]
I have the same problem (on Linux 64bit):-

**$ scide**
qt5ct: using qt5ct plugin
qt5ct: D-Bus global menu: no
Failed to open web socket server.
Aborted (core dumped)

**$ ps ax|grep scide** suggested that there was already a scide running (although no screen/window)

**$ pkill scide** killed the running version

**$ scide**
qt5ct: using qt5ct plugin
qt5ct: D-Bus global menu: no
qt5ct: palette support is disabled
...started correctly.

In conclusion, the hidden version of scide was blocking the web socket for another instance of the program to start and needed to be stopped (killed) to start another instance that shows correctly in a window.
